share/completions/keybase.fish

```diff
@@ -0,0 +1,77 @@
+#Keybase 5.0.0
+
+#variables
+set -l seen __fish_seen_subcommand_from
+set -l commands account blocks bot chat ctl currency decrypt deprovision device encrypt follow fs git help id list-followers list-following log login logout oneshot paperkey passphrase pgp ping prove rekey selfprovision service sign signup sigs status team track unfollow untrack update verify version wallet
+
+#global options
+#...
+
+#commands
+complete -c keybase -f -n "not $seen $commands" -a "$commands"
+#complete -c keybase -f -n "not $seen $commands" -a account -d "Modify your account"
```

Comment:
Any reason why these are all commented out?

Reply:
I had started adding option descriptions but there are too many so I wanted just to add the completions for now.